cloudwatch_metrics_collector/include/cloudwatch_metrics_collector/metrics_collector_parameter_helper.hpp

```diff
@@ -48,13 +53,28 @@ constexpr char kNodeParamMaximumFileSize[] = "maximum_file_size";
 constexpr char kNodeParamStorageDirectory[] = "storage_directory";
 constexpr char kNodeParamStorageLimit[] = "storage_limit";
 
+enum class TopicType
+{
+  ROS_MONITORING_MSGS = 1,
+  METRICS_STATISTICS_MSGS = 2
```

Comment:
We probably don't need to set 1 and 2 explicitly?

Reply:
Done.